pkg/loader/load.go

```diff
@@ -438,8 +452,30 @@ func (s *loaderImpl) execDMLs(dmls []*DML) error {
 	return errors.Trace(err)
 }
 
+func (s *loaderImpl) createMarkTableDDL() error {
```

Comment:
It seems the names of the two function `createMarkTableDDL` and `createMarkTable` should be swapped.